### DISCUSSION TITLE
Contributing design to Outline

### DISCUSSION THREAD

**Discussion by aaronang** [outline/outline#6913] [General]
Hi there! 👋 I was wondering if there's a way for designers to contribute to Outline. I noticed that there is a "design needed" label, but the only one seems to be from quite a while back (Feb 10, 2022). I'm curious if there are any current opportunities for open source designers to get involved with this project? Let me know if you have any guidance on that.
Thanks in advance for your help!

**Comment by tommoor** [comment]
Hey thanks - I think there are some areas that need more help than others, but of course you're welcome to experiment with anything.

One area I've always felt as rough as the sidebar panes - currently considering getting rid of insights here and just having comments and history with tabs at the top to switch between. The history pane itself is also very bland. 

One small warning is that we are constrained by engineering time more than anything so any designs might not get implemented straight away.

**Reply by aaronang** [reply]
I completely understand the engineering constraint. Are there any design assets available like Figma or Penpot? It's okay if it's not publicly available but it would make design contributions easier to iterate :)

**Reply by tommoor** [reply]
There aren't any designs in a design tool, these have always been done as one-offs working ontop of screenshots of the existing platform.
